### DISCUSSION TITLE
Reset to defaultColumns

### DISCUSSION THREAD

**Discussion by fabioquarantini** [payloadcms/payload#2419] [Feature Requests & Ideas]
Hi. It would be nice to be able to reset the default columns with a button.

**Comment by jmikrut** [comment]
That is a GREAT idea. We could build a button into the "Columns" dropdown that resets them quite easily. 

Are you interested in making a PR for this? We would happily accept! Otherwise I can slate it into the roadmap and get it done. Let me know!

**Reply by fabioquarantini** [reply]
I try to take a look. Thank you.
